quick_intro_pt1.rst

```diff
@@ -634,14 +643,15 @@ and ``Basket`` class looking like this:
         public function addProduct($product)
         {
             $this->products[] = $product;
-            $this->productsPrice += $this->shelf->getProductPrice($product);
+            $this->productsPrice += $this->shelf
+                ->getProductPrice($product);
```

Comment:
I'm not a big fan of this either. Which one should I do:

1. Put everything after `=` on new line with indentation
2. Leave it all on the same line (like it was before)